packages/react-dom/src/client/ReactDOMLegacy.js

```diff
@@ -32,17 +32,19 @@ import {
   findHostInstance,
   findHostInstanceWithWarning,
 } from 'react-reconciler/src/ReactFiberReconciler';
+import {isRendering as ReactCurrentFiberIsRendering} from 'react-reconciler/src/ReactCurrentFiber';
```

Comment:
This is a bit awkward. One alternative is to put `isRendering` onto `ReactDebugCurrentFrame` itself.